pennylane/templates/subroutines/all_singles_doubles.py

```diff
@@ -141,6 +141,9 @@ def __init__(self, weights, wires, hf_state, singles=None, doubles=None, id=None
         if weights_shape != exp_shape:
             raise ValueError(f"'weights' tensor must be of shape {exp_shape}; got {weights_shape}.")
 
+        # convert hf_state to numpy array in case it is a list
+        hf_state = np.array(hf_state) if isinstance(hf_state, list) else hf_state
```

Comment:
```suggestion
        hf_state = np.array(hf_state) if isinstance(hf_state, (list, tuple)) else hf_state
```

Reply:
BTW, it looks like all PRs are currently failing due to `catalyst`, so I'll commit the change above although the resulting PR will most probably fail